daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionCoderSpec.scala

```diff
@@ -271,7 +271,10 @@ class TransactionCoderSpec
 
         val normalized = normalizeNode(node) match {
           case exe: NodeExercises[NodeId, ContractId] =>
-            exe.copy(choiceObservers = node.choiceObservers)
+            exe.copy(
+              choiceObservers = node.choiceObservers,
+              exerciseResult = Some(Value.ValueText("not-missing")),
```

Comment:
we do not want to override existing `exerciseResult`, i.e. 
```suggestion
              exerciseResult = exe.exerciseResult.orEsle(Some(Value.ValueText("not-missing"))),
```